### PR TITLE
[Update] use link to replace fixed props

### PR DESCRIPTION
**Pull request by Icemap**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/11406
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: best-practices/java-app-best-practices.md [hunk 1]
```diff
@@ -198,15 +198,7 @@ In addition, because of a [client bug](https://bugs.mysql.com/bug.php?id=96623),
...
-`useConfigs=maxPerformance` configuration includes a group of configurations:
-
-```ini
-cacheServerConfiguration=true
-useLocalSessionState=true
-elideSetAutoCommits=true
-alwaysSendSetIsolation=false
-enableQueryTimeouts=false
-```
+`useConfigs=maxPerformance` configuration includes a group of configurations, check mysql-connector-j version [8.0](https://github.com/mysql/mysql-connector-j/blob/release/8.0/src/main/resources/com/mysql/cj/configurations/maxPerformance.properties) or [5.1](https://github.com/mysql/mysql-connector-j/blob/release/5.1/src/com/mysql/jdbc/configs/maxPerformance.properties) to confirm the specific configurations included in `maxPerformance` in the current MySQL JDBC.
```
Comment (Oreoxmt) [inline best-practices/java-app-best-practices.md:201]:
```suggestion
`useConfigs=maxPerformance` configuration includes a group of configurations. To get the detailed configurations in `maxPerformance` in the current MySQL JDBC, you can check mysql-connector-j version [8.0](https://github.com/mysql/mysql-connector-j/blob/release/8.0/src/main/resources/com/mysql/cj/configurations/maxPerformance.properties) or [5.1](https://github.com/mysql/mysql-connector-j/blob/release/5.1/src/com/mysql/jdbc/configs/maxPerformance.properties).
```

File: develop/dev-guide-connection-parameters.md [hunk 2]
```diff
@@ -275,15 +275,7 @@ In addition, because of a [client bug](https://bugs.mysql.com/bug.php?id=96623),
...
-`useConfigs=maxPerformance` configuration includes a group of configurations:
-
-```ini
-cacheServerConfiguration=true
-useLocalSessionState=true
-elideSetAutoCommits=true
-alwaysSendSetIsolation=false
-enableQueryTimeouts=false
-```
+`useConfigs=maxPerformance` configuration includes a group of configurations, check mysql-connector-j version [8.0](https://github.com/mysql/mysql-connector-j/blob/release/8.0/src/main/resources/com/mysql/cj/configurations/maxPerformance.properties) or [5.1](https://github.com/mysql/mysql-connector-j/blob/release/5.1/src/com/mysql/jdbc/configs/maxPerformance.properties) to confirm the specific configurations included in `maxPerformance` in the current MySQL JDBC.
```
Comment (Oreoxmt) [inline develop/dev-guide-connection-parameters.md:278]:
```suggestion
`useConfigs=maxPerformance` configuration includes a group of configurations. To get the detailed configurations in `maxPerformance` in the current MySQL JDBC, you can check mysql-connector-j version [8.0](https://github.com/mysql/mysql-connector-j/blob/release/8.0/src/main/resources/com/mysql/cj/configurations/maxPerformance.properties) or [5.1](https://github.com/mysql/mysql-connector-j/blob/release/5.1/src/com/mysql/jdbc/configs/maxPerformance.properties).
```

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: best-practices/java-app-best-practices.md [hunk 3]
```diff
@@ -198,15 +198,7 @@ In addition, because of a [client bug](https://bugs.mysql.com/bug.php?id=96623),
...
-`useConfigs=maxPerformance` configuration includes a group of configurations:
-
-```ini
-cacheServerConfiguration=true
-useLocalSessionState=true
-elideSetAutoCommits=true
-alwaysSendSetIsolation=false
-enableQueryTimeouts=false
-```
+`useConfigs=maxPerformance` configuration includes a group of configurations. To get the detailed configurations in `maxPerformance` in the current MySQL JDBC, you can check mysql-connector-j version [8.0](https://github.com/mysql/mysql-connector-j/blob/release/8.0/src/main/resources/com/mysql/cj/configurations/maxPerformance.properties) or [5.1](https://github.com/mysql/mysql-connector-j/blob/release/5.1/src/com/mysql/jdbc/configs/maxPerformance.properties).
```
Comment (shichun-0415) [inline best-practices/java-app-best-practices.md:201]:
```suggestion
`useConfigs=maxPerformance` includes a group of configurations. To get the detailed configurations in MySQL JDBC 8.0 and those in MySQL JDBC 5.1, see [mysql-connector-j 8.0](https://github.com/mysql/mysql-connector-j/blob/release/8.0/src/main/resources/com/mysql/cj/configurations/maxPerformance.properties) and [mysql-connector-j 5.1](https://github.com/mysql/mysql-connector-j/blob/release/5.1/src/com/mysql/jdbc/configs/maxPerformance.properties) respectively.
```

File: develop/dev-guide-connection-parameters.md [hunk 4]
```diff
@@ -275,15 +275,7 @@ In addition, because of a [client bug](https://bugs.mysql.com/bug.php?id=96623),
...
-`useConfigs=maxPerformance` configuration includes a group of configurations:
-
-```ini
-cacheServerConfiguration=true
-useLocalSessionState=true
-elideSetAutoCommits=true
-alwaysSendSetIsolation=false
-enableQueryTimeouts=false
-```
+`useConfigs=maxPerformance` configuration includes a group of configurations. To get the detailed configurations in `maxPerformance` in the current MySQL JDBC, you can check mysql-connector-j version [8.0](https://github.com/mysql/mysql-connector-j/blob/release/8.0/src/main/resources/com/mysql/cj/configurations/maxPerformance.properties) or [5.1](https://github.com/mysql/mysql-connector-j/blob/release/5.1/src/com/mysql/jdbc/configs/maxPerformance.properties).
```
Comment (shichun-0415) [inline develop/dev-guide-connection-parameters.md:278]:
```suggestion
`useConfigs=maxPerformance` includes a group of configurations. To get the detailed configurations in MySQL JDBC 8.0 and those in MySQL JDBC 5.1, see [mysql-connector-j 8.0](https://github.com/mysql/mysql-connector-j/blob/release/8.0/src/main/resources/com/mysql/cj/configurations/maxPerformance.properties) and [mysql-connector-j 5.1](https://github.com/mysql/mysql-connector-j/blob/release/5.1/src/com/mysql/jdbc/configs/maxPerformance.properties) respectively.
```

**Review by shichun-0415 — APPROVED** [review]
Rest LGTM

**Pull request merged**